clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1709,6 +1709,17 @@ with N banks.
   }];
 }
 
+def IntelFPGAMaxConcurrencyAttrDocs : Documentation {
+  let Category = DocCatVariable;
+  let Heading = "max_concurrency (IntelFGPA)";
+  let Content = [{
+This attribute may be attached to a variable or struct member declaration and
+instructs the backend to replicate the memory generated for the variable or
```

Comment:
`memory generated` might be unclear for people not familiar with FPGA.
But I do not now what to have instead. `memory synthesized`?

Reply:
> `memory generated` might be unclear for people not familiar with FPGA.
> But I do not now what to have instead. `memory synthesized`?

I agree.